dcrjson/chainsvrcmds.go

```diff
@@ -107,14 +107,43 @@ type EstimateFeeCmd struct {
 	NumBlocks int64
 }
 
-// NewEstimateFeeCmd returns a new instance which can be used to issue a
+// NewEstimateFeeCmd returns a new instance which can be used to issue an
 // estimatefee JSON-RPC command.
 func NewEstimateFeeCmd(numBlocks int64) *EstimateFeeCmd {
 	return &EstimateFeeCmd{
 		NumBlocks: numBlocks,
 	}
 }
 
+// EstimateSmartFeeMode defines estimation mode to be used with
+// the estimatesmartfee command.
+type EstimateSmartFeeMode int
```

Comment:
I think this should be a string as numbers change over time and adding new string modes is much less likely to break older clients.